cmd/provisioner-localpv/app/provisioner.go

```diff
@@ -125,6 +125,20 @@ func (p *Provisioner) Provision(opts pvController.VolumeOptions) (*v1.Persistent
 
 	glog.Infof("Creating volume %v at %v:%v", name, node.Name, path)
 
+	//Before using the path for local PV, make sure it is created.
+	initCmdsForPath := []string{"mkdir", "-m", "0777", "-p"}
+	podOpts := &HelperPodOptions{
+		cmdsForPath: initCmdsForPath,
+		name:        name,
+		path:        path,
+		nodeName:    node.Name,
+	}
+
+	if err := p.createInitPod(podOpts); err != nil {
```

Comment:
Wondering if building entire pod object should be done here instead of createInitPod.
Or perhaps have below functions:
- buildInitPod(b pod.Builder)
- applyAndCleanup(initPod)

Reply:
https://github.com/openebs/maya/issues/1228